node/declaration.ts

```diff
@@ -9,14 +9,19 @@ interface BaseDeclaration {
 
 type DeclarationWithPath = BaseDeclaration & {
   path: string
+  excludePath?: string
```

Comment:
All existing properties are nouns and "exclude path" is a verbal form. I wonder if we could use `excludedPath` or `negativePath` instead? 🤔

Reply:
Addressed in https://github.com/netlify/edge-bundler/pull/261/commits/775a0da5080b3be7df106fab88b9dd196de4eed9.